dist/api_config.py

```diff
@@ -156,7 +156,8 @@ def getconfcheck(c):
 w.wordsep_re = w.wordsep_simple_re = re.compile(r'(,)')
 
 # TextWrapper that wraps at whitespace.
-ws = textwrap.TextWrapper(width=64, break_on_hyphens=False)
+ws = textwrap.TextWrapper(width=64, break_on_hyphens=False,
+                          break_long_words=False)
```

Comment:
You should make this change to all TextWrapper instances in `api_config.py`

Reply:
If I add this setting to TextWrapper at line 127, it is resulting in longlines in wiredtiger.in. So, I haven't changed.

Reply:
That looks like a bug - it's splitting `start_generation` in the create API. Please open a ticket to track fixing that.